oslib_parser.py

```diff
@@ -1454,6 +1454,139 @@ def create_nvram_constants(defmods, filename):
                                 'values': values,
                             })
 
+def create_rust(defmods, dirname):
+    os.makedirs(os.path.join(dirname, "oslib", "src"), exist_ok=True)
```

Comment:
This has a slightly different pattern to the others, but not that unlike the cmodule generator.